_sass/_md.scss

```diff
@@ -1,4 +1,20 @@
 @media #{$md} {
+	.searchfield {
+		top: 82px;
+        width: 85vw;
+        max-width: 1100px;
```

Comment:
Sorry, this was left over from troubleshooting. I meant to set this to max width of nav menu plus max width of the content body. Both the nav menu and content body have set width (255 + 900px), so this should be 1155px. My intention was to avoid having the search box span the whole screen and instead only float over width of the nav menu and content body. I just thought it was odd to have the [Search] button so far off to the top left of the screen on its own.

Im fine with backing out my changes and letting it span the whole screen because I think a better solution might be to hide the search box into a icon (magnifying glass) in the banner so that we can gain some real estate back (without loosing its constant presence). Any thoughts here (this would be a backlog item)?